packages/http-client-csharp/emitter/src/options.ts

```diff
@@ -60,8 +60,8 @@ export const CSharpEmitterOptionsSchema: JSONSchemaType<CSharpEmitterOptions> =
     "generator-name": { type: "string", nullable: true },
     "emitter-extension-path": { type: "string", nullable: true },
     "update-code-model": { type: "object", nullable: true },
-    "namespace": { type: "string", nullable: true },
-    "license": {
+    namespace: { type: "string", nullable: true },
```

Comment:
FYI @schaabs, we want to avoid having to add this since we don't support it in unbranded.